source/docs/articles/wordpress/site-networks/getting-started.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Up and Running with WordPress Site Networks
+description: Get started with WordPress Site Networks. Learn about the Pantheon WordPress Site Network upstream product, start developing, or import existing networks.
+---
+
+Once a Pantheon employee has given you access to your WordPress Site Network site, you can import an existing site network or start from scratch and begin developing your network on the platform.
+
+## Migrating Site Networks to Pantheon
+
+If your network already exists, follow the instructions in the [Migrate to Pantheon: WordPress Site Networks](/docs/articles/sites/migrate/wordpress-site-networks/) doc. 
+
+## Create a Network
+Before you begin, switch the Development environment to SFTP mode. Creating the network requires making modifications to the `wp-config.php` file, which is not editable in Git mode.
+
```

Comment:
@bmackinney is that true? I can write changes to `wp-config.php` and push with Git - is there maybe a reason it shouldn't be done though? If so, we should update the Configuring wp-config.php doc to say why